apps/gp2-auth-frontend/scripts/build.js

```diff
@@ -0,0 +1,53 @@
+const { execSync } = require('child_process');
```

Comment:
technically this entire file is the same as https://github.com/yldio/asap-hub/blob/master/apps/crn-auth-frontend/scripts/build.js - i thought maybe you want to move it across to same shared package, although it might not be that easy since its a build script